### PR TITLE
Statoids - admin2 import for countries with good/great coverage

### PR DESCRIPTION
**Pull request by stepps00**
This PR includes updates at the `admin2` level. The updates include new `hasc:id` cocordance values and new properties harvested and created from Statoids data. Records in the following countries are updated with new Statoids data in this PR:

- Albania
- Argentina
- Bolivia
- Burundi
- Central African Republic
- Chile
- Dominican Republic
- El Salvador
- Equatorial Guinea
- Eritrea
- Gabon
- Georgia
- Ghana
- Guinea Bissau
- Iran
- Ivory Coast
- Kenya
- Kyrgyzstan
- Laos
- Malaysia
- Mali
- Mauritania
- Mongolia
- Morocco
- Mozambique
- New Zealand
- Nicaragua
- Nigeria
- Panama
- Republic of Congo
- Saudi Arabia
- Sierra Leone
- Somalia
- Swaziland
- Syria
- Tunisia

A [follow-up issue](LINK) has been opened to track any "leftover" records in these countries that were not updated in this PR. About 330 records in these countries share `hasc:id` concordances, so the follow-up issue will untangle those records, add new `hasc:id` concordances, and import Statoids data.

(A variant of) this script was used for this work: https://github.com/mapzen/whosonfirst-toolbox/blob/master/scripts/statoids_import_admin1.py

### CODE REVIEW

File: data/102/063/073/102063073.geojson [hunk 1]
```diff
@@ -26,6 +26,10 @@
     "mps:latitude":-24.413132,
     "mps:longitude":-69.54374,
     "mz:hierarchy_label":1,
+    "mz:is_current":-1,
```
Comment (nvkelso) [inline data/102/063/073/102063073.geojson]: I think this should be 1 for is valid?

Comment (stepps00) [inline data/102/063/073/102063073.geojson]: New commit incoming...

File: data/102/063/073/102063073.geojson [hunk 2]
```diff
@@ -41,6 +45,17 @@
...
     "src:lbl:centroid":"mapshaper",
+    "src:population":"statoids",
+    "statoids:areakm":"67,814",
+    "statoids:capital":"Antofagasta",
+    "statoids:country":"CL",
+    "statoids:date":"2012-04-09",
+    "statoids:hasc":"CL.AN",
+    "statoids:name":"Antofagasta",
+    "statoids:population":"318779",
+    "statoids:statoid":"CL.AN.AN",
```
Comment (nvkelso) [inline data/102/063/073/102063073.geojson]: I expected this `XX.YY.ZZ` code be stored in stations.hasc and to be stored as the has concordance. 

Or is HASC only for `region` level? In any event, we should be storing a concordance.

File: data/102/063/075/102063075.geojson [hunk 3]
```diff
@@ -26,6 +26,10 @@
     "mps:latitude":-52.448526,
     "mps:longitude":-71.062525,
     "mz:hierarchy_label":1,
+    "mz:is_current":-1,
+    "name:eng_x_preferred":[
+        "Magallanes"
```
Comment (nvkelso) [inline data/102/063/075/102063075.geojson]: Are you tracking variants that (now) match the preferred and marking them for removal? Please note &/or add that followup issue here.

File: data/102/063/073/102063073.geojson [hunk 4]
```diff
@@ -41,6 +45,17 @@
...
+    "statoids:timezone":"-4~",
```
Comment (nvkelso) [inline data/102/063/073/102063073.geojson]: Is there a followup issue to link up all the timezone information to our WOF timezone records in the `wof:belongsto`?

File: data/102/063/073/102063073.geojson (hunk 2 above)
Comment (stepps00) [inline data/102/063/073/102063073.geojson]: Statiods uses `hasc` to refer to the primary division (`admin1`) of a country, and `statoid` to refer to the secondary division (`admin2`) of a country.

For `admin2` records, the `stations:statoid` value is used as the `has:id` concordance.

File: data/110/856/452/1/1108564521.geojson [hunk 5]
```diff
@@ -49,10 +64,13 @@
         }
     ],
     "wof:id":1108564521,
-    "wof:lastmodified":1500939953,
+    "wof:lastmodified":1503612175,
     "wof:name":"Yagba East",
     "wof:parent_id":85675407,
     "wof:placetype":"county",
+    "wof:placetype_local":"l.g.a.",
```
Comment (nvkelso) [inline data/110/856/452/1/1108564521.geojson]: Please file followup task to translate `l.g.a.` to (I think) `local government authority`.

File: data/102/063/073/102063073.geojson (hunk 2 above)
Comment (nvkelso) [inline data/102/063/073/102063073.geojson]: Ah, it's already in the concordance list (which is why there is no diff to add it).

Comment (nvkelso) [inline data/102/063/073/102063073.geojson]: We should add a note to the sources repo to say `hasc:id` concordance has that variable property mapping.

Comment (stepps00) [inline data/102/063/073/102063073.geojson]: There are a handful of manually edited `hasc:id` properties that will appear in the diff, but the vast majority of the records in this PR had correct `hasc:id` values to begin with. 

New issue to update hasc source file here: https://github.com/whosonfirst/whosonfirst-sources/issues/84

File: data/110/856/452/1/1108564521.geojson (hunk 5 above)
Comment (stepps00) [inline data/110/856/452/1/1108564521.geojson]: Issue filed: https://github.com/whosonfirst-data/whosonfirst-data/issues/908

File: data/102/063/073/102063073.geojson (hunk 4 above)
Comment (stepps00) [inline data/102/063/073/102063073.geojson]: The `wof:belongsto` property is machine-generated and based on `wof:id` values in a record's `wof: hierarchy`; I'm not entirely sure the `wof: hierarchy` should include a `timezone_id`, but I've opened #909 to track updates (whatever they end up being) based on the `stations:timezone` property.

File: data/102/063/073/102063073.geojson (hunk 1 above)
Comment (stepps00) [reply data/102/063/073/102063073.geojson]: Taken care of in commit d3c5228fdb2445296746d93e8a779415e71c9212.

File: data/102/063/075/102063075.geojson (hunk 3 above)
Comment (stepps00) [reply data/102/063/075/102063075.geojson]: I'm not tracking these name issues, but I've opened https://github.com/whosonfirst-data/whosonfirst-data/issues/910 to track once the Statoids imports are complete.

**Comment by stepps00** [comment]
@nvkelso - all comments should be addressed, ready for another review/merge.

**Comment by thisisaaronland** [comment]
Placetypes need to be listed in `whosonfirst-placetypes` and need to follow the same naming conventions, which is basically lowercase a-z and underbars if necessary. Neither `l.g.a` or `local government authority` would be valid.

**Comment by stepps00** [comment]
Opened https://github.com/whosonfirst/whosonfirst-placetypes/issues/7 to update the `whosonfirst-placetypes` repo with new "local placetype" descriptions/files.

**Comment by nvkelso** [comment]
No, these are textual descriptions not enumerations of the WOF placetype.

**Comment by thisisaaronland** [comment]
Sorry - my comments were meant for: https://github.com/whosonfirst-data/whosonfirst-data/issues/908

Either way, I would prefer not to overload the meaning of the `wof:placetype_*` fields. Specifically, introducing a conditional that says these values always have a fixed, machine-readable value (`wof:placetype`, `wof:placetype_alt`) but these ones don't (`wof:placetype_local`).

We should definitely track local naming conventions but this feels like it might be better suited for the `label:` property.

**Pull request merged**